package/MDAnalysis/analysis/helix_analysis.py

```diff
@@ -306,6 +306,8 @@ class HELANAL(AnalysisBase):
         of best fit, and the local screw angles.
     flatten_single_helix : bool, optional
         Whether to flatten results if only one selection is passed.
+    split_residue_sequences : bool, optional
+        Wether to split residues into a list of same or consecutive helices.
```

Comment:
I'm not sure this is suitably informative for what it does. We should mention that this only is of use when `len(groups) > 1`. An example in the module docstring would be great if possible.

Reply:
Yes, this would be nice. However, I'm not familiar with the class... Maybe @lilyminium can help?

Reply:
Yes, let's do it this way.